addon/adapters/file-provider.js

```diff
@@ -0,0 +1,7 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
```

Comment:
Won't it return 'fileProviders'? The /files endpoint returns objects that should be treated as `file-provider`s, is the problem.